dts/bindings/flash_controller/nxp,iap-f1.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2019, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP (In-Application Programming) flash memory controller for lpc54114
```

Comment:
Can you reduce the length of this commit message title, so that it does not overflow into the commit body?